OpenOversight/tests/conftest.py

```diff
@@ -95,9 +100,9 @@ def generate_officer():
     )
 
 
-def build_assignment(officer, unit, jobs):
+def build_assignment(officer: Officer, unit: Unit, jobs: Job):
     return models.Assignment(star_no=pick_star(), job_id=random.choice(jobs).id,
-                             officer=officer)
+                             officer=officer, unit_id=unit.id, star_date=pick_date(), resign_date=pick_date())
```

Comment:
added unit and start/resign dates. The dates weren't strictly needed if they muck anything up